### ISSUE TITLE
user class

### ISSUE THREAD

**Issue by RheingoldRiver** [mwclient/mwclient#236]
One other thing I started working on was adding a class mwclient.user to make discovering and updating user rights easy, and maybe also support emailing users. Assuming I get around to finishing this, would you be interested in incorporating this to mwclient? Probably it would be sometime next month or the month after, I have a big project on my wiki I need to finish first, so far I only wrote a bit for a one-time set of user rights I needed to clone to people on a couple different wikis I work on.

**Comment by danmichaelo** [comment]
Absolutely! Note that we already have an `email()` method here: https://github.com/mwclient/mwclient/blob/master/mwclient/client.py#L479-L512

If you implement a user class, that method can probably be moved to the user class, leaving the original method as an alias (perhaps with a deprecation warning)
